### ISSUE TITLE
big error when `using NonExistentPackage`

### ISSUE THREAD

**Issue by haakon-e** [BioTurboNick/AbbreviatedStackTraces.jl#67]
possibly related to #27?

```julia
me@dhcp-72-190 dir % julia
               _
   _       _ _(_)_     |  Documentation: https://docs.julialang.org
  (_)     | (_) (_)    |
   _ _   _| |_  __ _   |  Type "?" for help, "]?" for Pkg help.
  | | | | | | |/ _` |  |
  | | |_| | | | (_| |  |  Version 1.9.3 (2023-08-24)
 _/ |\__'_|_|_|\__'_|  |  Official https://julialang.org/ release
|__/                   |

(@v1.9) pkg> st
Status `~/.julia/environments/v1.9/Project.toml`
  [ac637c84] AbbreviatedStackTraces v0.2.2
  [6e4b80f9] BenchmarkTools v1.3.2
  [89398ba2] LocalRegistry v0.5.5
  [85b6ec6f] MethodAnalysis v0.4.13
  [5fb14364] OhMyREPL v0.5.22
  [295af30f] Revise v3.5.6
  [fd094767] Suppressor v0.2.6
  [0c614874] TerminalPager v0.5.1

julia> using IDontExist
ERROR: ArgumentError: Package IDontExist not found in current path.
- Run `import Pkg; Pkg.add("IDontExist")` to install the IDontExist package.
SYSTEM (REPL): showing an error caused an error
ERROR: BoundsError: attempt to access 0-element Vector{Int64} at index [0]
Stacktrace:
  [1] getindex
    @ ./essentials.jl:13 [inlined]
  [2] find_visible_frames(trace::Vector{Any})
    @ AbbreviatedStackTraces ~/.julia/packages/AbbreviatedStackTraces/mBH2k/src/AbbreviatedStackTraces.jl:116
  [3] show_compact_backtrace(io::IOContext{Base.TTY}, trace::Vector{Any}; print_linebreaks::Bool)
    @ AbbreviatedStackTraces ~/.julia/packages/AbbreviatedStackTraces/mBH2k/src/AbbreviatedStackTraces.jl:187
  [4] show_compact_backtrace
    @ ~/.julia/packages/AbbreviatedStackTraces/mBH2k/src/AbbreviatedStackTraces.jl:135 [inlined]
  [5] show_backtrace(io::IOContext{Base.TTY}, t::Vector{Base.StackTraces.StackFrame})
    @ AbbreviatedStackTraces ~/.julia/packages/AbbreviatedStackTraces/mBH2k/src/override-errorshow.jl:61
  [6] showerror(io::IOContext{Base.TTY}, ex::ArgumentError, bt::Vector{Base.StackTraces.StackFrame}; backtrace::Bool)
    @ Base ./errorshow.jl:90
  [7] show_exception_stack(io::IOContext{Base.TTY}, stack::Base.ExceptionStack)
    @ AbbreviatedStackTraces ~/.julia/packages/AbbreviatedStackTraces/mBH2k/src/override-errorshow.jl:146
  [8] display_error(io::IOContext{Base.TTY}, stack::Base.ExceptionStack, compacttrace::Bool)
    @ AbbreviatedStackTraces ~/.julia/packages/AbbreviatedStackTraces/mBH2k/src/override-client.jl:25
  [9] #invokelatest#2
    @ ./essentials.jl:819 [inlined]
 [10] invokelatest
    @ ./essentials.jl:816 [inlined]
 [11] print_response(errio::IOContext{Base.TTY}, response::Pair{Any, Bool}, show_value::Bool, have_color::Bool, specialdisplay::Nothing)
    @ AbbreviatedStackTraces ~/.julia/packages/AbbreviatedStackTraces/mBH2k/src/override-REPL.jl:127
 [12] (::REPL.var"#57#58"{REPL.LineEditREPL, Pair{Any, Bool}, Bool, Bool})(io::Any)
    @ REPL /Users/julia/.julia/scratchspaces/a66863c6-20e8-4ff4-8a62-49f30b1f605e/agent-cache/default-honeycrisp-R17H3W25T9.0/build/default-honeycrisp-R17H3W25T9-0/julialang/julia-release-1-dot-9/usr/share/julia/stdlib/v1.9/REPL/src/REPL.jl:287
 [13] with_repl_linfo(f::Any, repl::REPL.LineEditREPL)
    @ REPL /Users/julia/.julia/scratchspaces/a66863c6-20e8-4ff4-8a62-49f30b1f605e/agent-cache/default-honeycrisp-R17H3W25T9.0/build/default-honeycrisp-R17H3W25T9-0/julialang/julia-release-1-dot-9/usr/share/julia/stdlib/v1.9/REPL/src/REPL.jl:557
 [14] print_response(repl::REPL.AbstractREPL, response::Any, show_value::Bool, have_color::Bool)
    @ REPL /Users/julia/.julia/scratchspaces/a66863c6-20e8-4ff4-8a62-49f30b1f605e/agent-cache/default-honeycrisp-R17H3W25T9.0/build/default-honeycrisp-R17H3W25T9-0/julialang/julia-release-1-dot-9/usr/share/julia/stdlib/v1.9/REPL/src/REPL.jl:285
 [15] (::REPL.var"#do_respond#80"{Bool, Bool, REPL.var"#93#103"{REPL.LineEditREPL, REPL.REPLHistoryProvider}, REPL.LineEditREPL, REPL.LineEdit.Prompt})(s::REPL.LineEdit.MIState, buf::Any, ok::Bool)
    @ REPL /Users/julia/.julia/scratchspaces/a66863c6-20e8-4ff4-8a62-49f30b1f605e/agent-cache/default-honeycrisp-R17H3W25T9.0/build/default-honeycrisp-R17H3W25T9-0/julialang/julia-release-1-dot-9/usr/share/julia/stdlib/v1.9/REPL/src/REPL.jl:899
 [16] #invokelatest#2
    @ ./essentials.jl:819 [inlined]
 [17] invokelatest
    @ ./essentials.jl:816 [inlined]
 [18] run_interface(terminal::REPL.Terminals.TextTerminal, m::REPL.LineEdit.ModalInterface, s::REPL.LineEdit.MIState)
    @ REPL.LineEdit /Users/julia/.julia/scratchspaces/a66863c6-20e8-4ff4-8a62-49f30b1f605e/agent-cache/default-honeycrisp-R17H3W25T9.0/build/default-honeycrisp-R17H3W25T9-0/julialang/julia-release-1-dot-9/usr/share/julia/stdlib/v1.9/REPL/src/LineEdit.jl:2647
 [19] run_frontend(repl::REPL.LineEditREPL, backend::REPL.REPLBackendRef)
    @ REPL /Users/julia/.julia/scratchspaces/a66863c6-20e8-4ff4-8a62-49f30b1f605e/agent-cache/default-honeycrisp-R17H3W25T9.0/build/default-honeycrisp-R17H3W25T9-0/julialang/julia-release-1-dot-9/usr/share/julia/stdlib/v1.9/REPL/src/REPL.jl:1300
 [20] (::REPL.var"#62#68"{REPL.LineEditREPL, REPL.REPLBackendRef})()
    @ REPL ./task.jl:514

julia> 
```

**Comment by BioTurboNick** [comment]
Probably just something stupid I did in the last update. I'll fix it shortly 🙂

**Comment by haakon-e** [comment]
thanks!

**Comment by BioTurboNick** [comment]
Fix is on the way to the registry, v0.2.3. Thanks for the report!
